### DISCUSSION TITLE
How can I set foldenable to false in dashboard?

### DISCUSSION THREAD

**Discussion by devswi** [folke/snacks.nvim#316] [Q&A] [answered]
![CleanShot 2024-12-16 at 16 13 27@2x](https://github.com/user-attachments/assets/236f76e3-8a5c-41d0-8c63-62207cae9810)

I am using `kevinhwang91/nvim-ufo` plugin to display fold text. But I could not disable folding in dashboard. I tried autocmd as shown below but it didn't work.

```lua
vim.api.nvim_create_autocmd("FileType", {
  pattern = { "snacks_dashboard" },
  callback = function()
    require("ufo").detach()
    vim.opt_local.foldenable = false
    vim.wo.foldenable = false
  end,
})
```

**Comment by folke** [comment, marked as answer]
Folding is already disabled on the dashboard, so it seems nvim-ufo enables it anyway

**Reply by devswi** [reply]
Thank you for your response, after search nvim-ufo repo, maybe [`provider_selector`](https://github.com/kevinhwang91/nvim-ufo/blob/95cbe2e99901135704aabdc8732d722cf68b12c9/doc/example.lua#L11) can solve my problome. 

``` lua
local ftMap = {
  vim = "indent",
  python = { "indent" },
  git = "",
  vue = { "treesitter", "indent" },
  snacks_dashboard = "",
}
```

Add `snacks_dashboard` filetype into `ftMap` and return an empty string. Thanks again
